<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>format</class>
 <widget class="QDialog" name="format">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>456</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>456</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="save_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Discard|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="format_label">
      <property name="text">
       <string>Result format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_rational">
      <property name="text">
       <string>Rational</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">format_group</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="button_decimal">
      <property name="text">
       <string>Decimal</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">format_group</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Precision</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>save_box</sender>
   <signal>accepted()</signal>
   <receiver>format</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_box</sender>
   <signal>rejected()</signal>
   <receiver>format</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="format_group"/>
 </buttongroups>
</ui>
